src/data/github/repo.rs

```diff
@@ -0,0 +1,36 @@
+use crate::errors::*;
+
+use miette::{miette, IntoDiagnostic};
+use url::Url;
+
+#[derive(Debug)]
```

Comment:
```suggestion
#[derive(Debug, Clone)]
```

I always kick myself later when i leave this off but obviously not important